lib/modules/versioning/ubuntu/distribution.ts

```diff
@@ -0,0 +1,42 @@
+import dataFiles from '../../../data-files.generated';
+
+export type UbuntuDistroInfo = Record<string, string>;
+
+// Data file generated with:
+// ubuntu-distro-info --all -f | sed -r 's/Ubuntu|"|LTS //g; s/([0-9]+.[0-9]+) /\1=/; s/.*/\L&/; s/(=[a-z]*) [a-z]*/\1/g; s/^[ \t]*//' | jo
+const dataFile = 'data/ubuntu-distro-info.json';
+
+const ubuntuDistroInfo: UbuntuDistroInfo = JSON.parse(
```

Comment:
what if json file does not exist? wrong path?

Reply:
you are right, "dataFIle" is misleading, as its is not referring to an actual fs file. but rather a key used to index into an auto generated map.

CI builds will fail if the file is missing, so its part of the project. so i guess we assume it exists.